core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -49,6 +52,7 @@ private[spark] class Slf4jSink(
   MetricsSystem.checkMinimalPollingPeriod(pollUnit, pollPeriod)
 
   val reporter: Slf4jReporter = Slf4jReporter.forRegistry(registry)
+    .outputTo(LoggerFactory.getLogger("org.apache.spark.metrics.sink.Slf4jSink"))
```

Comment:
For Slf4jSink.scala :-

Its not about class path or class level in package. It is just a name. Ex.
If you keep name like **Spark.log4j**, it would still work. ( and use the
same **Spark.log4j** in **log4j.properties**).  So it won't matter even if we
move class to some other folder or package.